### DISCUSSION TITLE
Purchasing documentation and respective stock items

### DISCUSSION THREAD

**Discussion by wolffam** [inventree/InvenTree#2879] [General]
I'm curious about the various workflows around this subject.
How do people currently use Inventree to associate purchasing information with particular stock items?

My workflow is a bit unique since I need to compile all of the purchasing information of all the components for a single assembly into a single document.  I could automate that process with the API but I'm curious about how you would typically get the purchasing information uploaded for a component stock item in the first place

**Comment by matmair** [comment]
I use purchase orders to continuously collect what needs to be bought and then order it once enough is collected. We then add the prices once the delivery shows up together with the delivery note (with the final prices on it) via an API script. That way all the stock has prices attached.

**Reply by wolffam** [reply]
Hey @matmair ,

To be more specific, I need to attach purchasing documentation (invoices) to each batch of stock items.  Do others share a similar workflow?

**Reply by matmair** [reply]
@wolfram there is an attachment section for orders
